content/docs/studio/get-started.md

```diff
@@ -1,31 +1,38 @@
 # Get Started with DVC Studio
 
 Here, we will walk you through a tutorial to use DVC Studio for collaboration on
-your ML projects. You will need access to a Github, Gitlab or Bitbucket account
+your ML projects. You will need access to a GitHub, GitLab or Bitbucket account
 which has access to the Git repositories you want to connect. DVC Studio creates
 views from repositories when you connect to them.
 
+The following video provides you a quick overview of DVC Studio.
+
+https://www.youtube.com/watch?v=hKf4twg832g
+
 ## What is a view?
 
 A _view_ is an interactive representation of the experiments run in your ML
 project. DVC Studio automatically identifies datasets, metrics and
-hyperparameters in your ML experiments. You can also add custom files with the
-required data (more on this later). Using this data, DVC Studio creates a
-summary view. This includes tables of all the metrics across all your
-experiments. You can also generate plots and compare experiments here.
+hyperparameters in your ML experiments. The data can either be in DVC
```

Comment:
data has a very specific meaning in ML, here the use of `data` is misleading

Reply:
changing 'The data' to 'These details'

Reply:
pls check other places

how about Values ...?

Reply:
"This info?"